client/ionic/public/assets/content

```diff
@@ -0,0 +1 @@
+../../../../content
```

Comment:
what is this doing? if it's to preserve the directory, can use `.gitkeep`

Reply:
This is a soft link.  It causes the top-level directory contents to both be embedded in the app and published at ENDPOINT/content.

Reply:
got ya 👍